src/image-copy/azext_imagecopy/create_target.py

```diff
@@ -37,6 +39,7 @@ def create_target_image(cmd, location, transient_resource_group_name, source_typ
                                    '--sku', 'Standard_LRS'],
                                   subscription=target_subscription)
 
+    logger.warning(format_cmd(cli_cmd))

```

Comment:
Looks like you're printing every command. What about doing it just 1 time inside of the run_cli_command function?
BTW, that function is already printing it out, but as a "debug" message... why do we need it twice?

Reply:
Almost. But I skip `wait_for_blob_copy_operation` because `storage blob show` is called repeatedly.
The reason I make the change is that I want to print CLI command that can be executed directly.